tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -2,7 +2,10 @@ import { Fields, ISnippetConfig } from "./type";
 import { IEnvelope } from "@microsoft/applicationinsights-common";
 import { IConfiguration, Snippet } from "@microsoft/applicationinsights-web";
 
-(function (win: Window, doc: Document, snipConfig: ISnippetConfig) {
+declare var cfg:ISnippetConfig;
+// please see expandMin in gruntfile for defining cfg values
```

Comment:
Nice, it's probably worth adding more details here as looking into the gruntfile (if you don't know it) is not obvious.

Add comments around why we are declaring `cfg` here and so that the generated code can be replace etc.